Dockerfile

```diff
@@ -25,13 +25,13 @@ RUN DEBIAN_FRONTEND=noninteractive dpkg-reconfigure locales \
 
 ENV LC_ALL C.UTF-8
 
-ARG ARCH=amd64
-ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.3/supercronic-linux-$ARCH \
-    SUPERCRONIC=supercronic-linux-$ARCH \
+ARG TARGETARCH
```

Comment:
How does `TARGETARCH` get populated? Is id thanks to [the `platform` line](https://github.com/stringer-rss/stringer/blob/5dd869fe2c850405e1903c42f9def98f37a0bb28/.github/workflows/build.yml#L50) in our Github action?